src/features/search/components/useLocationChoice.ts

```diff
@@ -12,11 +13,12 @@ export const useLocationChoice = (
   section: LocationType.PLACE | LocationType.EVERYWHERE | LocationType.AROUND_ME
 ): { Icon: React.FC<BicolorIconInterface>; label: string; isSelected: boolean } => {
   const { searchState } = useSearch()
+  const { position } = useGeolocation()
 
   if (section === LocationType.EVERYWHERE)
     return {
       Icon: Everywhere,
-      label: 'Partout',
+      label: position !== null ? 'Partout' : 'Me localiser',
```

Comment:
On préfère en général ne pas utiliser la négation lorsque c'est possible pour faciliter la lecture.
Du coup on aurait plutôt 
```suggestion
      label: position === null ? 'Me localiser' : 'Partout',
```